extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/util/FeatureDataUtils.java

```diff
@@ -76,4 +82,56 @@ else if (crs != null) {
 		}
 		return defaultCRSEntry;
 	}
+
+	public static String getAxis(
+			final CoordinateReferenceSystem crs ) {
+		final CoordinateSystem cs = crs == null ? null : crs.getCoordinateSystem();
+		if (cs != null && cs.getDimension() > 0) return cs.getAxis(
+				0).getDirection().name().toString();
+		return "";
```

Comment:
Do to the complexity of some of the CRS managed in geotools, I am not sure that we guarantee that the absence of an axis is an erroneous condition.  I interpret such cases as undefined or assumed.  Although I could test a hand full of CRS.  If they all define an axis, then we could go the route of the error.  Or we could just accept the undefined case.   Then, the question is whether an formal exception is necessary in these cases, a defined return value or , in this case, an innocuous return value.

Thoughts?